plutarch-trace/impl/Plutarch/Trace/Disable.hs

```diff
@@ -0,0 +1,17 @@
+module Plutarch.Trace.Disable (ptrace', ptrace, ptraceIfTrue, ptraceIfFalse) where
+
+import Plutarch.Bool (PBool)
+import Plutarch.Prelude
+import Plutarch.String (PString)
+
+ptrace' :: Term s (PString :--> a :--> a)
+ptrace' = phoistAcyclic $ plam $ \_ y -> y
```

Comment:
Dead code.

Reply:
Do you mean `ptrace'`? Or something else?